### PR TITLE
emit stream_title_changed when title differs from track name

### PR DESCRIPTION
**Pull request by kingosticks**
I've been using this change quite happily and seems worth it until we have proper stream type support.

### CODE REVIEW

**Comment by adamcik** [comment]
Either way, maybe just a point or minor release since it's been ages since there has been a release. So just put small things like this there and get whatever it adds up to out,

**Comment by jodal** [comment]
Any comments on the change itself?

**Comment by adamcik** [comment]
Nope, not really.

File: mopidy/core/actor.py [hunk 1]
```diff
@@ -131,13 +131,15 @@ def tags_changed(self, tags):
...
-        if 'title' in tags and tags['title'] and 'organization' in tags:
+        # TODO: this is a hack to make sure we don't emit stream title changes
+        # for plain tracks. We need a better way to decide if something is a
+        # stream.
+        if 'title' in tags and tags['title']:
             title = tags['title'][0]
-            self.playback._stream_title = title
-            CoreListener.send('stream_title_changed', title=title)
+            current_track = self.playback.get_current_track()
+            if current_track is not None and current_track.name != title:
```
Comment (eamanu) [inline mopidy/core/actor.py]: what about 

```
if current_track and current_track.name != title:
```
?

**Comment by kingosticks** [comment]
Any reason not to merge this as it is? I'm happy for it to sit in develop and wait for 3.0 if that's easier.

**Comment by kingosticks** [comment]
Actually, I need to update some [tests](https://github.com/mopidy/mopidy/blob/develop/tests/core/test_playback.py#L923).

**Comment by kingosticks** [comment]
Still needs a changelog entry but I need to know into which branch it's headed.

**Comment by jodal** [comment]
Target it at `release-2.2`. We can decide at release time if it is 2.2.4 or 2.3.0.

**Comment by kingosticks** [comment]
OK done.

**Comment by kingosticks** [comment]
Why is there no CI build triggered here?

**Comment by jodal** [comment]
I've turned on this CircleCI setting now:

> Build forked pull requests
>
> Run builds for pull requests from forks. CircleCI will automatically update the commit status shown on GitHub's pull request page.

Try force-pushing to trigger a build.

**Pull request merged**